### ISSUE TITLE
Problem launching Angular under Win10

### ISSUE THREAD

**Issue by superflystudio** [angular/angular-cli#9932]
Hey, I'm Chris from France,

I've just installed the node-v8.10.0-x64.msi, without problem...

My dev stuff is very well configured, no problem. (local dev server with
Wampserver64 v.3.1.1, php 7.2, Apache 2.4.27,
MySql 5.7.19,....)

SO, when I type "ng help" or for example "ng new testproject1" I have
the same message :

"Ce programme est bloqué par une stratégie de groupe. Pour plus
d'informations, contactez votre administrateur système."
(thos program is blocked by group strategy. For more details, please
contact your system admin...."

What the F..K ???? I'm the admin of my workstation, full rights, and for
sure it's the only and first program which
send me that warning....
I tested 10000 solutions , no way...

Some idea ???

Best regards,

Chris

**Comment by Brocco** [comment]
This seems like a problem with your node/npm and not with Angular CLI. 

Please have a look at the [fixing npm permissions page](https://docs.npmjs.com/getting-started/fixing-npm-permissions), [common errors page](https://docs.npmjs.com/troubleshooting/common-errors), [npm issue tracker](https://github.com/npm/npm/issues), or open a new issue if the problem you are experiencing isn't known.

**Comment by superflystudio** [comment]
Hi Mike, 

Very happy to read you !!! 

I'm going to try with the nvm install.... 

Thanks a lot, 

Chris 

Le 12.03.2018 22:44, Mike Brocchi a écrit :

> This seems like a problem with your node/npm and not with Angular CLI. 
> 
> Please have a look at the fixing npm permissions page [1], common errors page [2], npm issue tracker [3], or open a new issue if the problem you are experiencing isn't known. 
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [4], or mute the thread [5].

 

Links:
------
[1] https://docs.npmjs.com/getting-started/fixing-npm-permissions
[2] https://docs.npmjs.com/troubleshooting/common-errors
[3] https://github.com/npm/npm/issues
[4]
https://github.com/angular/angular-cli/issues/9932#issuecomment-372473062
[5]
